CHANGELOG.md

```diff
@@ -6,13 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [0.21.1] - 2021-09-09
+## [0.22.0] - 2021-09-17
 ### Security
 - Upgrades Openssl in Alpine to resolve CVE-2021-3711.
   [cyberark/conjur-authn-k8s-client#392](https://github.com/cyberark/conjur-authn-k8s-client/issues/392)
 - Upgrades Alpine to v3.14 to resolve CVE-2021-36159.
   [cyberark/conjur-authn-k8s-client#374](https://github.com/cyberark/conjur-authn-k8s-client/issues/374)
 
+### Added
```

Comment:
Can you add a commit message to the commit in which these lines were added that explains that these were actually released in 0.21.1 but were missed from the CHANGELOG so they're being released as part of 0.22.0 instead?